vm/vm.go

```diff
@@ -203,24 +202,6 @@ func (vm *Engine) validateWitnessV0(virtualTx *wire.MsgTx, inputIdx uint32,
 		return err
 	}
 
-	for _, witnessItem := range witness.TxWitness {
-		// Signatures can either be 64, with SIGHASH_DEFAULT, or 65
-		// bytes otherwise.
-		//
-		// TODO(roasbeef): remove? will go thru normal sig parse
```

Comment:
Cool, I think we also need to remove this erroneous check on the spec level.

Reply:
Yeah I think that was part of the original intent since we already use a 1-in-1-out everywhere. Though I guess one could sign the virtual transaction, but with sighash_none which would let someone else fill in the final details hypothetically.